cas-server-webapp/src/main/webapp/WEB-INF/cas-servlet.xml

```diff
@@ -144,9 +144,10 @@
   <bean id="loginFlowHandlerMapping" class="org.springframework.webflow.mvc.servlet.FlowHandlerMapping"
         p:flowRegistry-ref="loginFlowRegistry" p:order="2">
       <property name="interceptors">
-          <array value-type="org.springframework.web.servlet.HandlerInterceptor">
-              <ref bean="localeChangeInterceptor" />
-          </array>
+      <array value-type="org.springframework.web.servlet.HandlerInterceptor">
+          <ref bean="localeChangeInterceptor" />
+          <ref bean="loginRateLimiting" />
```

Comment:
Config update: add bean `loginRateLimiting` as `interceptors` in `loginFlowHandleMapping`